### PR TITLE
Adding Azure to Auth0 chart

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Adding Azure to the Auth0 URI chart

## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-azure-auth0-dbt-labs.vercel.app/docs/cloud/manage-access/set-up-sso-saml-2.0
- https://docs-getdbt-com-git-azure-auth0-dbt-labs.vercel.app/docs/cloud/manage-access/sso-overview

### CODE REVIEW

**Review by nataliefiann — APPROVED** [review]
Hiya @matthewshaver 

I've approved this for you and updated the branch.

Kind regards
Natalie

File: website/snippets/auth0-uri.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-| Region | dbt Cloud Access URL | Auth0 SSO URI &lt;YOUR_AUTH0_URI&gt; | Auth0 Entity ID &lt;YOUR_AUTH0_ENTITYID&gt;* |
-|--------|-----------------------|-------------------------------|----------------------------------------|
-| US multi-tenant   | cloud.getdbt.com      | auth.cloud.getdbt.com | us-production-mt                       |
-| US cell 1 | \{account prefix\}.us1.dbt.com | auth.cloud.getdbt.com | us-production-mt |
-| EMEA   | emea.dbt.com         | auth.emea.dbt.com     | emea-production-mt                     |
-| APAC   | au.dbt.com           | auth.au.dbt.com       | au-production-mt                       |
+| Region              | dbt Cloud Access URL | Auth0 SSO URI &lt;YOUR_AUTH0_URI&gt; | Auth0 Entity ID &lt;YOUR_AUTH0_ENTITYID&gt;* |
+|---------------------|-----------------------|-------------------------------|----------------------------------------------------|
+| AWS NA multi-tenant | cloud.getdbt.com      | auth.cloud.getdbt.com         | us-production-mt                                   |
+| AWS NA cell 1       | \{account prefix\}.us1.dbt.com | auth.cloud.getdbt.com| us-production-mt                                   |
```
Comment (ajaythomas) [inline website/snippets/auth0-uri.md]: We now have 3 cells. So maybe rename this row as `AWS NA cell 1,2,3...`

File: website/snippets/auth0-uri.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
 
 The URI used for SSO connections on multi-tenant dbt Cloud instances will vary based on your dbt Cloud hosted region. Use your login URL (also referred to as your Access URL) to determine the correct Auth0 URI for your environment.
```
Comment (ajaythomas) [inline website/snippets/auth0-uri.md]: This is not for multi-tenants only. Even single-tenants will be using this table.

**Review by ajaythomas — CHANGES_REQUESTED** [review]

File: website/snippets/auth0-uri.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 
-The URI used for SSO connections on multi-tenant dbt Cloud instances will vary based on your dbt Cloud hosted region. Use your login URL (also referred to as your Access URL) to determine the correct Auth0 URI for your environment.
+The URI used for SSO connections on multi-tenant dbt Cloud instances will vary based on your dbt Cloud hosted region. To find the URIs for your environment in dbt Cloud:
```
Comment (ajaythomas) [inline website/snippets/auth0-uri.md:2]: > SSO connections on multi-tenant dbt Cloud instance

not just MT

**Review by ajaythomas — APPROVED** [review]

**Pull request merged**